test/integration/admin/api/services/backend_usages_controller_test.rb

```diff
@@ -144,15 +144,15 @@ def setup
     test 'it cannot create for a deleted backend_api' do
       backend_api.mark_as_deleted!
 
-      post admin_api_service_backend_usages_path(collection_params), {path: 'foo/bar', backend_api_id: backend_api.id}
+      post admin_api_service_backend_usages_path(collection_params), params: { path: 'foo/bar', backend_api_id: backend_api.id }
       assert_response :not_found
     end
   end
 
   class WithMemberAccessToken < self
     def setup
       super
-      @member = FactoryBot.create(:member, account: tenant, admin_sections: %w[partners])
+      @member = FactoryBot.create(:member, account: tenant, admin_sections: %w[partners plans]) # FIXME: it should not require 'partners' permission
```

Comment:
This is because we don't have a `plans` scope for the Admin API
https://github.com/3scale/porta/blob/f6f7a12ad20fb847b82397004eba3cfec8e39882/app/models/access_token.rb#L21

thus the `account_management` scope is mapped to the `partners` permission
https://github.com/3scale/porta/blob/f6f7a12ad20fb847b82397004eba3cfec8e39882/app/models/access_token.rb#L28

As a consequence, a user that has in the UI only the "Integration & Application Plans" permission (i.e. `plans`), because the corresponding endpoints in the Admin API are all scoped under the "Account Management" scope, requires his/her privileges to be raised to include as well the "Accounts -- Applications" (i.e. `partners`) permission, both in the UI and API, only so the can perform "Integration & Application Plans"-related actions in the API.